lib/commands/query.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+const { resolve } = require('path')
+const Arborist = require('@npmcli/arborist')
+const BaseCommand = require('../base-command.js')
+
+class QuerySelectorItem {
+  constructor (node) {
```

Comment:
```suggestion
  constructor ({
    target: {
      pkgid: this.pkgid
      location: this.location,
      path: this.path,
      realpath: this.realpath,
      resolved: this.resolved,
      isLink: this.isLink,
      isWorkspace: this.isWorkspace,
    },
  }) {
```
i know this is a bit horrific but i think it does work :-p

Reply:
I think I'm gonna leave it as is.